abi.html

```diff
@@ -5171,6 +5171,10 @@ <h4><a href="#expressions">5.1.6 Expressions</a></h4>
                ::= sZ &lt;<a href="#mangle.function-param">function-param</a>&gt;                                  # sizeof...(parameter), size of a function parameter pack
                ::= sP &lt;<a href="#mangle.template-arg">template-arg</a>&gt;* E                                 # sizeof...(T), size of a captured template parameter pack from an alias template
                ::= sp &lt;<a href="#mangle.expression">expression</a>&gt;                                      # expression..., pack expansion
+               ::= fl &lt;<i>binary</i> <a href="#mangle.operator-name">operator-name</a>&gt; &lt;<a href="#mangle.expression">expression</a>&gt;               # ... operator expression, unary left fold
+               ::= fr &lt;<i>binary</i> <a href="#mangle.operator-name">operator-name</a>&gt; &lt;<a href="#mangle.expression">expression</a>&gt;               # expression operator ..., unary right fold
+               ::= fL &lt;<i>binary</i> <a href="#mangle.operator-name">operator-name</a>&gt; &lt;<a href="#mangle.expression">expression</a>&gt; &lt;<a href="#mangle.expression">expression</a>&gt;  # expression operator ... operator expression, binary left fold
+               ::= fR &lt;<i>binary</i> <a href="#mangle.operator-name">operator-name</a>&gt; &lt;<a href="#mangle.expression">expression</a>&gt; &lt;<a href="#mangle.expression">expression</a>&gt;  # expression operator ... operator expression, binary right fold
```

Comment:
These are missing the parens in the comments.

Reply:
Right you are.  Fixed now.